tools/install-powershell.ps1

```diff
@@ -135,6 +135,10 @@ try {
         $downloadURL = "https://github.com/PowerShell/PowerShell/releases/download/v${release}/${packageName}"
         Write-Verbose "About to download package from '$downloadURL'" -Verbose
 
+        # Setting Tls to 12 to prevent the Invoke-WebRequest : The request was
+        # aborted: Could not create SSL/TLS secure channel. error.
+        [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
I believe we should use the same from #6236 
[Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor [Net.SecurityProtocolType]::Tls12

Reply:
Yes.. and for `Install-` the original setting needs to be reverted. after the calls.